### PR TITLE
Fix some of the Flutter logic

### PR DESCRIPTION
**Pull request by rodydavis**

### CODE REVIEW

File: dataconnect/lib/profile.dart [hunk 1]
```diff
@@ -31,13 +31,6 @@ class _ProfileState extends State<Profile>
   @override
   void initState() {
     super.initState();
-    Auth.getCurrentUser().then((user) {
```
Comment (maneesht) [inline dataconnect/lib/profile.dart:34]: Why was this deleted?

Comment (rodydavis) [reply dataconnect/lib/profile.dart:34]: Fixed! I forgot to add the authguard there. I was thinking the user came from the future
